src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.services.DataConverter;
+import core.basesyntax.services.DataConverterImp;
+import core.basesyntax.services.OperationHandler;
+import core.basesyntax.services.ReaderService;
+import core.basesyntax.services.ReaderServiceImp;
+import core.basesyntax.services.ReportGenerator;
+import core.basesyntax.services.ReportGeneratorImp;
+import core.basesyntax.services.StorageService;
+import core.basesyntax.services.StorageServiceImp;
+import core.basesyntax.services.TransactionProcessor;
+import core.basesyntax.services.WriterService;
+import core.basesyntax.services.WriterServiceImp;
+import core.basesyntax.services.operations.BalanceOperation;
+import core.basesyntax.services.operations.PurchaseOperation;
+import core.basesyntax.services.operations.ReturnOperation;
+import core.basesyntax.services.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_FILE = "src/main/resources/reportToRead.csv";
+    private static final String OUTPUT_FILE = "src/main/resources/finalReport.csv";
+
+    public static void main(String[] args) {
+        try {

```

Comment:
1) keep only the potentially vulnerable part of the code in the try block
2) exception catching is not needed here, let's remove it